documentation/ipi-install/modules/ipi-install-node-requirements.adoc

```diff
@@ -3,40 +3,39 @@
 // * installing/installing_bare_metal_ipi/ipi-install-prerequisites.adoc
 
 [id='node-requirements_{context}']
+
 = Node requirements
 
 IPI installation involves a number of hardware node requirements:
 
-- **CPU architecture:** All nodes *must* use `x86_64` CPU architecture.
+- **CPU architecture:** All nodes must use `x86_64` CPU architecture.
```

Comment:
I think this is bolded downstream, do we not want to keep this emphasis?

Reply:
It's based on https://github.com/openshift/openshift-docs/blob/enterprise-4.6/modules/ipi-install-node-requirements.adoc#L11

Reply:
Ashley asked me to remove it per the style guide.